pkg/clients/edgeconnect/config.go

```diff
@@ -5,5 +5,5 @@ import (
 )
 
 var (
-	log = logger.Factory.GetLogger("edgeConnectClient")
+	log = logger.Factory.GetLogger("edgeConnectClient") //nolint: unused
```

Comment:
why not using it ? (then just remove it)

Reply:
logger is unused, removed